src/olympia/addons/tests/test_commands.py

```diff
@@ -322,3 +326,62 @@ def test_populate_e10s_feature_compatibility_with_unlisted():
     assert addon_compatible_unlisted.feature_compatibility.pk
     assert (addon_compatible_unlisted.feature_compatibility.e10s ==
             amo.E10S_COMPATIBLE_WEBEXTENSION)
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'pre_addon_status, pre_file1_status, pre_file2_status, enabled, '
+    'end_addon_status, end_file1_status, end_file2_status, end_experimental, '
+    'added_note, send_email',
+    [(amo.STATUS_LITE, amo.STATUS_LITE, amo.STATUS_LITE, True,
+      amo.STATUS_PUBLIC, amo.STATUS_PUBLIC, amo.STATUS_PUBLIC, True,
+      True, True),
+     (amo.STATUS_LITE, amo.STATUS_LITE, amo.STATUS_LITE, False,
+      amo.STATUS_PUBLIC, amo.STATUS_PUBLIC, amo.STATUS_PUBLIC, True,
+      True, False),
+     (amo.STATUS_UNREVIEWED, amo.STATUS_DISABLED, amo.STATUS_UNREVIEWED, True,
+      amo.STATUS_NOMINATED, amo.STATUS_DISABLED, amo.STATUS_UNREVIEWED, True,
+      True, True),
+     (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_LITE, amo.STATUS_LITE, True,
+      amo.STATUS_NOMINATED, amo.STATUS_DISABLED, amo.STATUS_UNREVIEWED, False,
+      False, False),
+     (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_LITE, amo.STATUS_UNREVIEWED,
+      True,
+      amo.STATUS_NOMINATED, amo.STATUS_DISABLED, amo.STATUS_UNREVIEWED, False,
+      False, False),
+     (amo.STATUS_PUBLIC, amo.STATUS_LITE, amo.STATUS_PUBLIC, True,
+      amo.STATUS_PUBLIC, amo.STATUS_DISABLED, amo.STATUS_PUBLIC, False,
+      False, False)])
+def test_migrate_preliminary(
+        pre_addon_status, pre_file1_status, pre_file2_status, enabled,
+        end_addon_status, end_file1_status, end_file2_status, end_experimental,
+        added_note, send_email, mozilla_user):
```

Comment:
`mozilla_user` is a function further up the file that just creates the `TASK_USER_ID` user so `amo.log` doesn't baulk.